common/src/types.ts

```diff
@@ -1176,6 +1206,7 @@ export interface JudgementLookup {
 	"ddr:SP": DDRJudges;
 	"ddr:DP": DDRJudges;
 	"maimai:Single": "good" | "great" | "miss" | "perfect";
+	"maimaidx:Single": "good" | "great" | "miss" | "pcrit" | "perfect";
```

Comment:
swap pcrit and perfect so they're in the right order?

Reply:
it's listed in alphabetical order, if i looked at the other entries correctly.

Reply:
The linter enforces alphabetical order for unions for sane git diffs. it looks ugly in scenarios like these, sadly